src/wiki/models/article.py

```diff
@@ -206,13 +207,20 @@ def get_cache_key(self):
             lang=lang
         )
 
-    def get_cached_content(self):
+    def get_cached_content(self, user=None):
         """Returns cached version of rendered article"""
+        user_key = str(user) if user else ""
         cache_key = self.get_cache_key()
-        cached_content = cache.get(cache_key)
+        cache_content_key = "{}:{}".format(cache_key, user_key)
```

Comment:
I agree that the cache key should contain a user component, but should this logic be moved to `self.get_cache_key` and then let the method accept a `user` kwarg?

Reply:
fixed